extension/js/common/platform/catch.ts

```diff
@@ -311,8 +312,13 @@ export class Catch {
 
   private static async doSendErrorToSharedTenantFes(errorReport: ErrorReport) {
     try {
-      const uncheckedUrlParams = Url.parse(['acctEmail']);
-      const acctEmail = String(uncheckedUrlParams.acctEmail);
+      let acctEmail = Url.parse(['acctEmail']).acctEmail;
+      if (acctEmail) {
+        acctEmail = String(acctEmail);
+      } else {
+        const acctEmails = await GlobalStore.acctEmailsGet();
+        acctEmail = acctEmails?.length ? acctEmails[0] : undefined;
+      }
```

Comment:
How about write like below?

```ts
const { acctEmail: parsedEmail } = Url.parse(['acctEmail']);
let acctEmail = parsedEmail 
    ? String(parsedEmail) 
    : (await GlobalStore.acctEmailsGet())?.[0];

```

Reply:
I think that is why we added safe casting (?) before accessing first element?

> ?.[0]

Reply:
Yeah, you're right - I thought `?` checks only for `undefined` or `null` values (similar to Swift behavior), but it works for empty arrays too. I updated code according to your suggestion, thanks!